guides/release/components/built-in-components.md

```diff
@@ -98,11 +99,17 @@ Checkboxes support the following properties:
 Which can be bound or set as described in the previous section.
 
 
-Checkboxes are a special input type. If you want to dispatch an action on a certain [event](https://api.emberjs.com/ember/3.11/classes/Component#event-handler-methods), you will always need to define the event name in camelCase format:
+Checkboxes are a special input type.  If you want to dispatch an action on a certain [event](https://api.emberjs.com/ember/release/classes/Component#event-handler-methods), you will always need to define the event name in [Web-API format](https://developer.mozilla.org/en-US/docs/Web/API/Document/keydown_event):
+Or per the camelCase mapping in Application.customEvents in packages/@ember/-internals/views/lib/system/event_dispatcher.js
```

Comment:
just showing my work. should be removed before considered for merge. but I'm just not sure whether the guide is still correct or not